flocker/node/agents/test/test_blockdevice.py

```diff
@@ -169,7 +173,12 @@
 
 DESIRED_DATASET_ATTRIBUTE_STRATEGIES = {
     'dataset_id': uuids(),
-    'maximum_size': integers(min_value=0).map(
+    'maximum_size': integers(
+        min_value=0,
+        max_value=(
+            sys.maxint - LOOPBACK_MINIMUM_ALLOCATABLE_SIZE
+        ) // LOOPBACK_ALLOCATION_UNIT,
```

Comment:
I'm not at all sure about this.
Intent is to calculate a max int which will be <= maxint after it's plugged into the sum below....but I'm not sure about the sum below either.